### ISSUE TITLE
将控制台MySQL消息由ERROR级改为INFO级

### ISSUE THREAD

**Issue by ZoruaFox** [HaHaWTH/AuthMeReReloaded#193]
### What behaviour is observed?

根据观察，在使用MySQL作为数据源时，开启与关闭连接log会被标记为ERROR，如下：
```java
[12:50:11 ERROR]: [AuthMe] [Server thread] INFO fr.xephi.authme.libs.com.zaxxer.hikari.HikariDataSource - AuthMeMYSQLPool - Starting...
[12:50:11 ERROR]: [AuthMe] [Server thread] INFO fr.xephi.authme.libs.com.zaxxer.hikari.HikariDataSource - AuthMeMYSQLPool - Start completed.
```

### Expected behaviour

日志以INFO等级输出

### To Reproduce

使用本fork插件即可

### Plugin list

Unnecessary

### Server Implementation

Standalone(Spigot)

### Database Implementation

MySQL

### AuthMe Version

AuthMeReloaded v5.7.0-FORK (build: 52)

### Configuration

Unnecessary

**Comment by HaHaWTH** [comment]
能否在AuthMeReloaded上复现

**Comment by ZoruaFox** [comment]
> 能否在AuthMeReloaded上复现

测试环境缺失暂未查证，等待其他用户

**Comment by tester43214** [comment]
> 能否在AuthMeReloaded上复现

Yes it is the same on AuthMe Reloaded v5.6.0-SNAPSHOT-b2599 version.

[18:40:31] [Server thread/ERROR]: [AuthMe] [STDERR] [Server thread] INFO fr.xephi.authme.libs.com.zaxxer.hikari.HikariDataSource - AuthMeMYSQLPool - Starting...
[18:40:31] [Server thread/WARN]: Nag author(s): '[sgdc3, ljacqu, games647, Hex3l, krusic22]' of 'AuthMe v5.6.0-SNAPSHOT-b2599' about their usage of System.out/err.print. Please use your plugin's logger instead (JavaPlugin#getLogger).
[18:40:32] [Server thread/ERROR]: [AuthMe] [STDERR] [Server thread] INFO fr.xephi.authme.libs.com.zaxxer.hikari.HikariDataSource - AuthMeMYSQLPool - Start completed.

**Comment by rautamiekka** [comment]
Why do you people even care about those lines ?

^ The SQL ones are giving you useful info (in this case, things are working as to be expected), and the "`Nag author(s)`" is just a necessary evil due to how the system works, which AuthMe (or Hikari, the SQL library) can't avoid (I tried but couldn't find where someone like a dev said pretty much that, which I recall reading): https://github.com/AuthMe/AuthMeReloaded/issues/2832 https://github.com/AuthMe/AuthMeReloaded/issues/2539

^ You're finding a problem where no problem exists.
